guides/pro/authorization.md

```diff
@@ -93,8 +93,8 @@ end
 Also, you can specify authentication at __type-level__, for example:
 
 ```ruby
-AccountBalanceType = GraphQL::ObjectType.define do
-  name "AccountBalance"
+class AccountBalanceType < GraphQL::ObjectType
+  graphql_name "AccountBalance"
```

Comment:
In class-based syntax, the default behavior is to remove `Type` from the end of the class name, then use that for `graphql_name`. So, I think this line is unnecessary in the new API, does that sound right to you? 
```suggestion
```